src/utils/rest-api.js

```diff
@@ -1254,7 +1254,7 @@ export function fetchDynamicSimulationResultTimeSeries(
         timeSeriesNames.length > 0 &&
         '?' + getQueryParamsList(timeSeriesNames, 'timeSeriesNames');
 
-    url += paramsList && '';
+    url += paramsList ?? '';
```

Comment:
paramsList cannot be undefined or null, so it is possible to remove entirely the "` ?? ''; `"

Reply:
In fact, the back end accepts also request without param. In this case back end will return all series.
This is an API so every cases must be covered.

Reply:
Ok, my comment was incorrect, I thought these two evaluations would both return false : 

```
false && true && '?' + 'hello'
=> Returns false

undefined && true && '?' + 'hello'
=> Returns undefined
```